loadbalancer-standalone/src/main/java/com/rtbhouse/grpc/loadbalancer/standalone/AvailableServerList.java

```diff
@@ -0,0 +1,225 @@
+package com.rtbhouse.grpc.loadbalancer.standalone;
+
+import static java.lang.Math.ceil;
+
+import com.google.protobuf.ByteString;
+import io.grpc.lb.v1.Server;
+import io.grpc.lb.v1.ServerList;
+import java.net.InetSocketAddress;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class AvailableServerList {
```

Comment:
Nazwa tej klaski jest nieco myląca, bo to nie jest zwykła lista, a raczej taki zarządzający listą dostępnych serwerów managerek.

Reply:
To już raczej subiektywne, nie chciałbym jej zmieniać bo już się do niej przyzwyczailiśmy.